client/components/payment-methods-checkboxes/index.js

```diff
@@ -0,0 +1,11 @@
+/** @format */
```

Comment:
Mostly the same as WCPay's https://github.com/Automattic/woocommerce-payments/tree/develop/client/components/payment-methods-checkboxes.